physics/cires_ugwp_triggers_v1.F90

```diff
@@ -0,0 +1,576 @@
+module cires_ugwp_triggers_v1
+
+
+contains
+
+
+        subroutine ugwp_triggers
+        implicit none
+        write(6,*) ' physics-based triggers for UGWP ' 
```

Comment:
Not sure what this does ...

Reply:
Can we leave this as-is until Valery has a look?

Reply:
Ok. If it clutters the model stdout we'll have to change it.